contrib/opentimelineio_contrib/adapters/ale.py

```diff
@@ -116,6 +117,33 @@ def _parse_data_line(line, columns, fps):
                 target_url=source
             )
 
+        # If available, collect cdl values in the same way we do for CMX EDL
+        cdl = {}
+
+        if 'CDL' in metadata:
+            cdl = _cdl_values_from_metadata(metadata['CDL'])
+            if cdl:
+                metadata.pop('CDL')
+
+        # If we have more specific metadata, let's use them
+        if 'ASC_SOP' in metadata:
+            cdl = _cdl_values_from_metadata(metadata['ASC_SOP'])
+
+            if cdl:
+                metadata.pop('ASC_SOP')
+
+        if 'ASC_SAT' in metadata:
+            if metadata['ASC_SAT']:
```

Comment:
You could combine these two ifs into one like this: `if metadata.get('ASC_SAT')` since `get` will return `None` if the key is missing.

Reply:
Yup, I generally use the `in` keyword because it should perform in constant time since it performs an hashing function, while I don't know if get() internally tries to get the key and catches a KeyError (which is generally slower). But yeah, no harm in just using `get` I guess!